From 0d41ca480b30746adceffe1a161e9f64fd77bf36 Mon Sep 17 00:00:00 2001
From: Vivek Goyal <vgoyal@redhat.com>
Date: Tue, 19 Aug 2014 22:02:43 -0400
Subject: [crypto] keys/x509: Fix a spelling mistake

Message-id: <1408485785-777-12-git-send-email-vgoyal@redhat.com>
Patchwork-id: 88365
O-Subject: [RHEL7.1 PATCH 11/33] KEYS: X.509: Fix a spelling mistake
Bugzilla: 1129779
RH-Acked-by: David Howells <dhowells@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

BZ:1129779

commit 452069867c1dcfe9cb60fd89bb1c92f5ec2fb3c8
Author: David Howells <dhowells@redhat.com>
Date: Tue, 8 Jul 2014 17:21:01 +0100

Signed-off-by: David Howells <dhowells@redhat.com>
Acked-by: Vivek Goyal <vgoyal@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/crypto/asymmetric_keys/Kconfig b/crypto/asymmetric_keys/Kconfig
index eb36225..d73c3a5 100644
--- a/crypto/asymmetric_keys/Kconfig
+++ b/crypto/asymmetric_keys/Kconfig
@@ -32,7 +32,7 @@ config X509_CERTIFICATE_PARSER
  select ASN1
  select OID_REGISTRY
  help
-   This option procides support for parsing X.509 format blobs for key
+   This option provides support for parsing X.509 format blobs for key
    data and provides the ability to instantiate a crypto key from a
    public key packet found inside the certificate.
 
-- 
1.7.1